machine_management/creating_machinesets/creating-machineset-nutanix.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="creating-machineset-nutanix"]
+= Creating a machine set on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: creating-machineset-nutanix
+
+toc::[]
+
+You can create a different machine set to serve a specific purpose in your {product-title} cluster on Nutanix. For example, you might create infrastructure machine sets and related machines so that you can move supporting workloads to the new machines.
```

Comment:
```suggestion
You can create a different machine set to serve a specific purpose in your {product-title} cluster on Nutanix. For example, you might create infrastructure machine sets and related machines so you can move supporting workloads to the new machines.
```

Reply:
I agree, but that's stock text used all over the place. I'll open a ticket to change all instances at once and backport it to all applicable versions :+1: